### PR TITLE
troubleshooting procedure

### PR DESCRIPTION
**Pull request by tomaslin**
surface places where people might look for more details when they run into a bug

### CODE REVIEW

**Comment by tomaslin** [comment]
@duftler - PTAL

File: documentation/troubleshooting.md [hunk 1]
```diff
@@ -10,6 +10,68 @@ lang: en
...
+|Rush|8085|
+
+
+### Check Service Logs ###
+
+If you have identified a service, you can check the service logs.
+
+If you used one of the pre-baked Spinnaker machine images or installed Spinnaker from the .deb files, each subsystem will write its logs to:
+
+`/var/log/spinnaker/{subsystem-name}/{subsystem-name}.log`
```
Comment (duftler) [inline documentation/troubleshooting.md]: In the other pending PR I replaced 'subsystem' with 'service'.

File: documentation/troubleshooting.md [hunk 2]
```diff
@@ -10,6 +10,68 @@ lang: en
...
+
+![API](../images/troubleshooting/api.png)
+
+###Check Health / Config endpoints###
+
+Most Spinnaker services have a health check endpoint configured. Sometimes this can provide useful information in terms of visualizing service health. To see if there is any information available, click on http://[service url]/health (e.g, [http://localhost:8084/health](http://localhost:8084/health)).
+
+Sometimes issues arise from incorrectly set environment variables or configuration files. You can see configurations available by click on http://[service url]/env (e.g, [http://localhost:8084/env](http://localhost:8084/env)), which will provide a dump of the service environment. 
+
+There are a few other endpoints available actuator endpoints can be seen in the [Spring Boot Actuator Endpoints documentation.](http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/#production-ready).
```
Comment (duftler) [inline documentation/troubleshooting.md]: This renders with two trailing periods.

File: documentation/troubleshooting.md [hunk 3]
```diff
@@ -10,6 +10,68 @@ lang: en
...
+
+To access the API, go to http://[gate url]/swagger/index.html ( for example [http://localhost:8084/swagger/index.html](http://localhost:8084/swagger/index.html) )
+
+![API](../images/troubleshooting/api.png)
+
+###Check Health / Config endpoints###
+
+Most Spinnaker services have a health check endpoint configured. Sometimes this can provide useful information in terms of visualizing service health. To see if there is any information available, click on http://[service url]/health (e.g, [http://localhost:8084/health](http://localhost:8084/health)).
+
+Sometimes issues arise from incorrectly set environment variables or configuration files. You can see configurations available by click on http://[service url]/env (e.g, [http://localhost:8084/env](http://localhost:8084/env)), which will provide a dump of the service environment.
```
Comment (duftler) [inline documentation/troubleshooting.md]: s/click/clicking

File: documentation/troubleshooting.md [hunk 4]
```diff
@@ -10,6 +10,68 @@ lang: en
...
+
+![API](../images/troubleshooting/api.png)
+
+###Check Health / Config endpoints###
+
+Most Spinnaker services have a health check endpoint configured. Sometimes this can provide useful information in terms of visualizing service health. To see if there is any information available, click on http://[service url]/health (e.g, [http://localhost:8084/health](http://localhost:8084/health)).
+
+Sometimes issues arise from incorrectly set environment variables or configuration files. You can see configurations available by click on http://[service url]/env (e.g, [http://localhost:8084/env](http://localhost:8084/env)), which will provide a dump of the service environment. 
+
+There are a few other endpoints available. A list is available on the [Spring Boot Actuator Endpoints documentation.](http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/htmlsingle/#production-ready).
```
Comment (duftler) [inline documentation/troubleshooting.md]: Don't know if my comment survived the last commit. This link renders with two trailing periods.

**Comment by tomaslin** [comment]
@duftler please review again

**Comment by duftler** [comment]
LGTM.

**Pull request merged**